micrometer-jersey2/src/main/java/io/micrometer/jersey2/server/MetricsRequestEventListener.java

```diff
@@ -93,6 +95,9 @@ public void onEvent(RequestEvent event) {
                     }
                 }
 
+                registry.summary(metricName + ".request.size", tagsProvider.httpRequestTags(event)).record(containerRequest.getLength());
+                registry.summary(metricName + ".response.size", tagsProvider.httpRequestTags(event)).record(containerResponse.getLength());
```

Comment:
There was a comment on the original issue about whether to record the size of the body and/or headers. If we tag this as the body, then an instrumentation can add the values for headers if that's useful for someone. I'm not sure what the best tag name would be though... maybe `http.part` or even just `part`, with the value in this case being `body`

Reply:
I believe `http.headers.size` could work.

Not sure about actually measure the size (in bytes) of headers though, at it could lead to some security/performance issues, since there is no good way to obtain it.

Unrelated to headers size, should response size collection be optional maybe something like `autoSizeRequests` and `@Sized`?

Reply:
Any opinions?